app/controllers/v2/link_changes_controller.rb

```diff
@@ -1,7 +1,7 @@
 module V2
   class LinkChangesController < ApplicationController
     def index
-      render json: Queries::GetLinkChanges.new(params).as_json
+      render json: Queries::GetLinkChanges.new(params, request.original_url).as_json
```

Comment:
I wonder if we have scope to re-use the keyset pagination code that was wrote for the editions endpoint? https://github.com/alphagov/publishing-api/blob/master/app/controllers/v2/editions_controller.rb#L3-L30

I know @thomasleese intended for it to be reusable and if this endpoint is quite consistent with that this could be a good chance for consistent approaches.

Reply:
Yeah, I agree it would be good to use the keyset pagination code here. Probably easier to explain off GitHub though, so let me know if you want more background on how you could do this.